### PR TITLE
Set multiple as done

### PR DESCRIPTION
**Pull request by jooleyaz**
Added function that sets multiple wishes as purchased, given an array of wishids

### CODE REVIEW

File: backend/src/handlers/wishHandler.ts [hunk 1]
```diff
@@ -35,35 +35,48 @@ async function EditWish(wish: Partial<Wish>): Promise<void> {
...
     }
 };
 
-export { getAWish, DeleteWish, EditWish, CreateWish }
\ No newline at end of file
+// takes in a string of wishids, marks them all as purchased aka complete
+async function markMultipleWishesAsDone(wishes: String[]): Promise<void> {
+    const client = await Pool.connect();
+    try {
+        console.log(wishes);
```
Comment (jaykbpark) [inline backend/src/handlers/wishHandler.ts]: please remove console.log

File: backend/src/handlers/wishHandler.ts [hunk 2]
```diff
@@ -35,35 +35,48 @@ async function EditWish(wish: Partial<Wish>): Promise<void> {
...
+        for (let wishid of wishes) {
+            const result = await client.query(`UPDATE wish SET purchased = true WHERE wishid = $1`, [wishid]);
```
Comment (jaykbpark) [inline backend/src/handlers/wishHandler.ts]: try {
        await client.query('BEGIN');
        const query = 'UPDATE wish SET purchased = true WHERE wishid = ANY($1)';
        await client.query(query, [wishIds]);
        await client.query('COMMIT');
    } catch (err) {
        await client.query('ROLLBACK');
        throw new Error("Issue with marking multiple wishes as done: " + err);
    } finally {
        client.release();
    }

**Review by jaykbpark — CHANGES_REQUESTED** [review]
plz review

**Comment by jooleyaz** [comment]
@jaypark25 REVIEW!!!

**Review by jaykbpark — APPROVED** [review]
lgtm!

**Pull request merged**